module/Motion/CorrectingRotation.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file   CorrectingRotation.h
+ * @brief  角度補正回頭
+ * @author aridome222
+ */
+
+#ifndef CORRECTING_ROTATION_H
+#define CORRECTING_ROTATION_H
+
+#include "Motion.h"
+#include "Mileage.h"
+#include "Timer.h"
+#include "SpeedCalculator.h"
+#include "StringOperator.h"
+#include "SystemInfo.h"
+
+class CorrectingRotation : public Motion {
+ public:
+  /**
+   * コンストラクタ
+   * @param _targetAngle 目標角度(deg) 0~89
+   * @param _targetSpeed 目標速度[mm/s]
+   */
+  CorrectingRotation(int _targetAngle, double _targetSpeed);
+
+  /**
+   * @brief 角度補正回頭する
+   */
+  void run();
+
+  /**
+   * @brief 実行のログを取る
+   */
+  void logRunning();
```

Comment:
```suggestion
  /**
   * @brief 角度補正回頭する際の事前条件判定をする
   * @param _targetAngle 目標角度(deg) 0~89
   * @param _targetSpeed 目標速度[mm/s]
   * @note オーバーライド必須
   */
  bool isMetPrecondition(int targetAngle, double targetSpeed) override;
  
  /**
   * @brief 実行のログを取る
   * @note オーバーライド必須
   */
  void logRunning() override;
```
CompositeMotion 継承なので、logRunning はオーバーライドが必要。事前条件も同様。